watermark/watermark.py

```diff
@@ -350,4 +354,4 @@ def _get_gpu_info():
                 "to be installed on this machine."}
     except:
         return {"GPU Info": "GPU information is not "
-                "available for this machine."}
\ No newline at end of file
+                "available for this machine."}
```

Comment:
So in that case, I'd say if `py3nvml = None`, we should probably then show a different message. I guess most users will interpret 

> "GPU information is not available for this machine."

as a GPU not being present or the GPU not being detected.

If someone uses the gpu flag and `py3nvml = None`, we should probably show something like

> "Please install the py3nvml dependency (`pip install py3nvml`) to display GPU information."